Anchor.toml

```diff
@@ -49,33 +49,23 @@ startup_wait = 20000
 [test.validator]
 url = "https://api.mainnet-beta.solana.com"
 
-[[test.genesis]]
+[[test.validator.clone]]
```

Comment:
Too many clones slows boot time and sometimes causes timeouts, any reason to stop building from deps?

Reply:
Interesting. Reverted.